annotated_text/util.py

```diff
@@ -136,7 +136,21 @@ def get_annotated_html(*args):
         elif isinstance(arg, tuple):
             out(annotation(*arg))
 
+        elif isinstance(arg,list):
+            for el in arg:
+                if isinstance(el, str):
+                    out(html.escape(el))
+
+                elif isinstance(el, HtmlElement):
+                    out(el)
+
+                elif isinstance(el, tuple):
+                    out(annotation(*el))
```

Comment:
Just stumbled upon this PR by pure chance and I'm just wondering, is there any reason why you rewrote the function instead of recursively calling it? `get_annotated_html(*list)` seems like it would do the work.